### ISSUE TITLE
Bool should inherit from Integer not Object

### ISSUE THREAD

**Issue by deep110** [pybee/voc#486]
In CPython, bool extends from int, but so is not the case in voc. Does this need fix or is it an intended implementation.

**Comment by freakboy3742** [comment]
VOC should be a complete mirror of Python's behaviour - so yes, this means inheritance relationships should be identical, too. 

However, keep in mind that this means there needs to be a *python* inheritance chain, which doesn't *necessarily* mean the Java Bool class needs to inherit from Int.

**Comment by auscompgeek** [comment]
Having Bool inherit from Int would likely simplify the implementation, however.

**Comment by uranusjr** [comment]
@auscompgeek Yes, and that applies not only to bool and int, but also a lot of other types. bytes/bytearray comes to mind, float and int share similarities, and many collection classes likely do as well. But those are just implementation, and can be refactored at any time.
